File: app/src/main/java/com/zarudna/foodplacesnearby/injections/DatabaseModule.java
package com.zarudna.foodplacesnearby.injections;

import dagger.Module;
import dagger.Provides;
import io.realm.Realm;

@Module
public class DatabaseModule {

    @Provides
    Realm providesRealm() {
        return Realm.getDefaultInstance();
    }
}

File: app/src/main/java/com/zarudna/foodplacesnearby/injections/DatabaseModule.kt
package com.zarudna.foodplacesnearby.injections;

import dagger.Module;
import dagger.Provides;
import io.realm.Realm;

@Module
class DatabaseModule {

    @Provides
    fun providesRealm(): Realm {
        return Realm.getDefaultInstance();
    }
}
